lib/trento/discovery/payloads/cluster/cluster_discovery_payload.ex

```diff
@@ -176,6 +176,28 @@ defmodule Trento.Discovery.Payloads.Cluster.ClusterDiscoveryPayload do
     end)
   end
 
+  defp parse_cluster_additional_sids(%{
+         "cluster_type" => ClusterType.hana_scale_up(),
+         "cib" => %{"configuration" => %{"resources" => %{"primitives" => primitives}}}
+       }) do
+    primitives
```

Comment:
Could we create a new function like `get_sapinstance_sids` or similar?
To avoid having the exact same code duplicated twice.